src/system-node.js

```diff
@@ -23,8 +23,6 @@ export function applyImportMap(loader, newMap, mapBase) {
   loader[IMPORT_MAP_PROMISE] = Promise.resolve();
 }
 
-export { clearFetchCache } from './features/node-fetch.js';
```

Comment:
Removing this is technically a breaking change. However, I seriously doubt people will have already started using this.

If we want to avoid the breaking change, we could make this be a no-op function for now.